plugin/src/main/java/com/playmonumenta/epicstructures/commands/CompassRespawn.java

```diff
@@ -26,18 +26,21 @@ public static void register(Plugin plugin) {
 			.withPermission(perms)
 			.withArguments(arguments)
 			.executes((sender, args) -> {
-				forceRespawn(sender, plugin, (String)args[0]);
+				if (sender instanceof Player) {
+					forceRespawn(sender, plugin);
+				}
 			})
 			.register();
 	}
 
-	private static void forceRespawn(CommandSender sender, Plugin plugin, String label) {
+	private static void forceRespawn(CommandSender sender, Plugin plugin) {
 		if (plugin.mRespawnManager == null) {
 			return;
 		}
 		try {
 			Player player = (Player) sender;
-			if (player.hasMetadata("ForceReset")) {
+			if (player.hasMetadata("ForceResetPOI")) {
+				String label = (String)player.getMetadata("MetadataKey").get(0).value();
 				plugin.mRespawnManager.compassRespawn(player, label);
 			}
```

Comment:
Maybe an else statement here with the the error message here about "Waited too long"?